board/pine64/ox64/boot-m1s.cmd

```diff
@@ -0,0 +1,7 @@
+echo "Loading Kernel from SD card"
+mmc rescan
+load mmc 0:2 0x50200000 Image
+echo "This kernel is configure for Sipeed M1S board"
+echo "please see /root/readme.txt to learn how to switch to other boards"
+load mmc 0:2 0x51ff8000 bl808-sipeed-m1s.dtb
+booti 0x50200000 - 0x51ff8000
```

Comment:
I should get the Linux DTB changes merged into U-Boot, so we can use `$fdtcontroladdr` for the DTB here instead of loading it from the SD card -- then the filesystem image would be board-agnostic. Or at least have U-Boot export `$fdtfile` for the file name.

Reply:
Im happy to iterate over this as we make progress. Just ping me when its done :)

Reply:
Not sure if you saw, but I also had to put a few mmc rescan commands in the default boot command...otherwise on fresh reboots I'd get messages about the card not responding to voltage change or something.